config/elasticsearch/mappings/feed_content.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
I love that we have this unspoken/understood rule going of ordering the mappings with `id` and then the rest of the properties in alphabetical order 💯